src/concrete/ml/torch/hybrid_model.py

```diff
@@ -190,9 +190,8 @@ def init_fhe_client(
                 path_dir=str(path_to_client.resolve()), key_dir=str(self.path_to_keys.resolve())
             )
             # The client first need to create the private and evaluation keys.
-            client.generate_private_and_evaluation_keys()
-            # Get the serialized evaluation keys
-            serialized_evaluation_keys = client.get_serialized_evaluation_keys()
+            serialized_evaluation_keys = client.generate_private_and_evaluation_keys()
```

Comment:
I was thinking more the other way 😅  like keep ` serialized_evaluation_keys = client.get_serialized_evaluation_keys()` but include `self.generate_private_and_evaluation_keys()` in `get_serialized_evaluation_keys` 

or then have a new method called "generate_keys_and_get_serialized_evaluation_keys` or similar

but that was more of an idea, we could make it an issue and consider it later if you prefer

Reply:
Yeah actually it makes more sense the way you describe. Will update.